src/main/java/org/openmaptiles/layers/Transportation.java

```diff
@@ -702,6 +726,19 @@ public List<VectorTile.Feature> postProcess(int zoom, List<VectorTile.Feature> i
       }
     }
 
+    //Upgrade small trunk segments at z5 so they can merge with surrounding motorways
+    if (zoom == 5) {
+      for (var item : items) {
+        var highway = item.tags().get(Fields.CLASS);
+        if (highway instanceof String highwayStr && highwayStr.equals(FieldValues.CLASS_TRUNK)) {
```

Comment:
Minor: can simplify a bit to:

```suggestion
        if (FieldValues.CLASS_TRUNK.equals(highway)) {
```